app/views/govuk_publishing_components/components/layout_header/_navigation_items.html.erb

```diff
@@ -1,7 +1,8 @@
+<%  navigation_items_label ||= "Top level" %>
```

Comment:
This default value for `navigation_items_label` could be here or in the `_layout_header` template. What's the benefit of having it here?

Reply:
Hm I think I added this here to provide a fallback in case `navigation_items_label` is ever not passed to `_navigation_items.html.erb`. But in hindsight I can probably get around this by adding a conditional around the `aria-label` where it's not added unless `navigation_items_label` exists. I will update the PR with that adjustment.

Reply:
Have updated so that the default value for `navigation_aria_label` is now set in `_layout_header.html.erb`.